src/Stryker.sln

```diff
@@ -17,6 +17,11 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Stryker.RegexMutators", "St
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Stryker.RegexMutators.UnitTest", "Stryker.RegexMutators\Stryker.RegexMutators.UnitTest\Stryker.RegexMutators.UnitTest.csproj", "{7C8C7670-4636-4C56-9E3E-22B987337EF2}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{68DC3E8E-7CC2-4953-BAFC-08EFBF9F3307}"
+	ProjectSection(SolutionItems) = preProject
+		..\azure-pipelines.yml = ..\azure-pipelines.yml
```

Comment:
Is this necessary? If so, why only the main pipeline file?

Reply:
Well no it's not necessary. I'm just used to the pipeline file being included in the solution. But we're missing the templates now. I could add them too?

Reply:
Could you then also add the .editorconfig, Directory.Build.props and Directory.Build.targets? They are currently referenced in the Stryker.CLI project but it makes more sense as a solution level item imo. Perhaps even the docs folder as well?

Reply:
I can't add a whole folder... Adding individual docs files could work but we will surely forget to add new doc files in the future

Reply:
That is a shame. In that case don't add them.